app/services/internal_record_importer_service.rb

```diff
@@ -0,0 +1,31 @@
+class InternalRecordImporterService
+  def self.get_request(vendor_name:, start_date:, end_date:)
```

Comment:
`start_date` and `end_date` should either pad the date here or in the csv importer